doc/sphinx/changes.rst

```diff
@@ -8,6 +8,675 @@ Recent changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  In :tacn:`refine`, new existential variables unified with existing ones are no
+  longer considered as fresh. The behavior of :tacn:`simple refine` no longer depends on
+  the orientation of evar-evar unification problems, and new existential variables
+  are always turned into (unshelved) goals. This can break compatibility in
+  some cases (`#7825 <https://github.com/coq/coq/pull/7825>`_, by Matthieu
+  Sozeau, with help from Maxime Dénès, review by Pierre-Marie Pédrot and
+  Enrico Tassi, fixes `#4095 <https://github.com/coq/coq/issues/4095>`_ and
+  `#4413 <https://github.com/coq/coq/issues/4413>`_).
+- **Changed:** Heuristics for universe minimization to :g:`Set`: also
+  use constraints ``Prop <= i`` (`#10331
+  <https://github.com/coq/coq/pull/10331>`_, by Gaëtan Gilbert with
+  help from Maxime Dénès and Matthieu Sozeau, fixes `#12414
+  <https://github.com/coq/coq/issues/12414>`_).
+
+  .. _813VarianceDecl:
+
+- **Added:** Commands :cmd:`Inductive`, :cmd:`Record` and synonyms now
+  support syntax `Inductive foo@{=i +j *k l}` to specify variance
+  information for their universes (in :ref:`Cumulative <cumulative>`
+  mode) (`#12653 <https://github.com/coq/coq/pull/12653>`_, by Gaëtan
+  Gilbert).
+- **Changed:**
+  Tweaked the algorithm giving default names to arguments.
+  Should reduce the frequency that argument names get an
+  unexpected suffix.
+  Also makes :flag:`Mangle Names` not mess up argument names.
+  (`#12756 <https://github.com/coq/coq/pull/12756>`_,
+  fixes `#12001 <https://github.com/coq/coq/issues/12001>`_
+  and `#6785 <https://github.com/coq/coq/issues/6785>`_,
+  by Jasper Hugunin).
+
+  .. _813UnusedVar:
+
+- **Added:**
+  Warning on unused variables in pattern-matching branches of
+  :n:`match` serving as catch-all branches for at least two distinct
+  patterns.
+  (`#12768 <https://github.com/coq/coq/pull/12768>`_,
+  fixes `#12762 <https://github.com/coq/coq/issues/12762>`_,
+  by Hugo Herbelin).
+- **Removed:**
+  Undocumented and experimental forward class hint feature ``:>>``.
+  Use ``:>`` (see :n:`@of_type`) instead
+  (`#13106 <https://github.com/coq/coq/pull/13106>`_,
+  by Pierre-Marie Pédrot).
+- **Fixed:**
+  Implicit arguments taken into account in defined fields of a record type declaration
+  (`#13166 <https://github.com/coq/coq/pull/13166>`_,
+  fixes `#13165 <https://github.com/coq/coq/issues/13165>`_,
+  by Hugo Herbelin).
+- **Added:**
+  Definition and (Co)Fixpoint now support the :attr:`using` attribute.
+  It has the same effect as :cmd:`Proof using`, which is only available in
+  interactive mode.
+  (`#13183 <https://github.com/coq/coq/pull/13183>`_,
+  by Enrico Tassi).
+- **Removed:** The type given to :cmd:`Instance` is no longer automatically
+  generalized over unbound and :ref:`generalizable <implicit-generalization>` variables.
+  Use :n:`Instance : \`{@type}` instead of :n:`Instance : @type` to get the old behaviour, or
```

Comment:
This attempt to escape a backtick leads to the backslash being kept in the rendered HTML:

![2020-12-02-125240_703x98_scrot](https://user-images.githubusercontent.com/1108325/100869402-4bbb2d80-349d-11eb-81a9-6db96344864e.png)

It's weird, because I didn't find any other documented way of escaping backticks.
@cpitclaudel Can you help?

Reply:
I fixed this using verbatim text, it works :)